### PR TITLE
Fixes to the howto-k8s-crosscluster walkthrough

### PR DESCRIPTION
**Pull request by herbertgoto**
*Issue #, if available:*

*Description of changes:*

Minor fixes to README to make it more intuitive. 
Fix a broken step when defining a LB Service. It requires a targetPort. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: walkthroughs/howto-k8s-cross-cluster/README.md [hunk 1]
```diff
@@ -51,24 +51,29 @@ Note that this walkthrough assumes throughout to operate in the us-east-1 region
```
Comment (Y0Username) [inline walkthroughs/howto-k8s-cross-cluster/README.md:51]: Can you remove L50?

File: walkthroughs/howto-k8s-cross-cluster/README.md [hunk 2]
```diff
@@ -51,24 +51,29 @@ Note that this walkthrough assumes throughout to operate in the us-east-1 region
...
 ```
-eksctl create cluster --name=eksc2 --nodes=3 --alb-ingress-access 
---region=us-east-1 --ssh-access --asg-access  --full-ecr-access  
---external-dns-access --appmesh-access --vpc-cidr 172.16.0.0/16
+eksctl create cluster --name=eksc2 --nodes=3 --alb-ingress-access \
+--region=$AWS_DEFAULT_REGION --ssh-access --asg-access  --full-ecr-access \
+--external-dns-access --appmesh-access --vpc-cidr 172.16.0.0/16 \
 --auto-kubeconfig
-#[✔]  EKS cluster "eksc2-useast1" in "us-east-1" region is ready
+#[✔]  EKS cluster "eksc2-useast1" is ready
```
Comment (Y0Username) [inline walkthroughs/howto-k8s-cross-cluster/README.md:65]: nit: `eksc2-useast1` -> `eksc2`

**Comment by Y0Username** [comment]
@achevuru looks like your change https://github.com/aws/aws-app-mesh-examples/pull/316 pinned the region to `us-east-1`, any specific reason for that?

File: walkthroughs/howto-k8s-cross-cluster/README.md (hunk 1 above)
Comment (herbertgoto) [reply walkthroughs/howto-k8s-cross-cluster/README.md]: Done

**Comment by achevuru** [comment]
> @achevuru looks like your change #316 pinned the region to `us-east-1`, any specific reason for that?

No specific reason, it was an existing walkthrough which we modified to make it work with GA version of the controller. No req to limit it to us-east-1

**Comment by Y0Username** [comment]
Thanks for the review @achevuru 
@herbertgoto could you please rebase this?

**Comment by herbertgoto** [comment]
@Y0Username I tested in different regions and it worked. I also eliminated the x-ray instructions, since x-ray sidecar is not installed anywhere in this workshop and those traces do not appear in x-ray.

**Review by Y0Username — APPROVED** [review]
Thanks @herbertgoto

**Pull request merged**